src/lib/style/components/_popover.scss

```diff
@@ -56,4 +56,5 @@
 
 .PopoverContainer {
   display: inline-block;
+  cursor: pointer;
```

Comment:
Clicks on any children of our MarkdownPopover will cause the popover to show, so I changed the cursor to reflect that.